eve-api-data/src/test/java/com/eveonline/api/repo/AccountBalanceTests.java

```diff
@@ -20,10 +16,9 @@
 	@Test
 	@Sql({"/testGetAccountId34339379Balance.sql"})
 	public void testGetAccountId34339379Balance() {
-		AccountBalance balance = repo.findByAccountKeyAndCorporationId(1000, 1l);
-		BigDecimal expected = new BigDecimal(200000.35);
-		expected = expected.setScale(2, BigDecimal.ROUND_HALF_UP);
-		assertEquals("Incorrect balance for corp and account key", expected, balance.getBalance());
+		AccountBalance balance = repo.findByAccountKeyAndCorporationId(1000, 1);
+		//BigDecimal expected = new BigDecimal(200000.35);
```

Comment:
This was a hint that some type issue's would arise when the unit test was executed.